app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -61,7 +61,7 @@ def link_identity_to_service_provider
     def handle_successful_handoff
       track_events
       SpHandoffBounce::AddHandoffTimeToSession.call(sp_session)
-      redirect_to @authorize_form.success_redirect_uri
+      redirect_to @authorize_form.success_redirect_uri, allow_other_host: true
```

Comment:
should this have gotten caught by our linter before?

Reply:
the linter was only checking for `redirect_back` because it uses the Referer header, which makes it vulnerable to open redirects

Reply:
ahhh I see thanks! 👍

Reply:
this must be a new default for `redirect_to`, which is nice